keyboards/bt66tech/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
+OPT_DEFS = -DCORTEX_VTOR_INIT=0x2000
+MCU_LDSCRIPT = STM32F103x8_stm32duino_bootloader
+BOARD = STM32_F103_STM32DUINO
+
+DFU_ARGS = -d 1eaf:0003 -a2 -R
+DFU_SUFFIX_ARGS = -v 1eaf -p 0003
+
+# project specific files
+VPATH += keyboards/bt66tech/bluepill
+SRC =	keyboard.c
```

Comment:
If you rename the file to `bluepill.c` it should be picked up, by default. 

Otherwise, those files may be best being moved to the bt66tech.c file.